sys/net/link_layer/Kconfig.lorawan

```diff
@@ -524,4 +523,21 @@ config LORAMAC_DEFAULT_MIN_RX_SYMBOLS
         system timer. Refer SX1276_settings_for_LoRaWAN_v2p2.pdf
         (AN1200.24) from Semtech for more information.
 
+config LORAMAC_DEFAULT_ADR_ACK_LIMIT
+    int "ACK limit (ADR)" if LORAMAC_ADVANCED_OPTIONS
+    depends on USEMODULE_GNRC_LORAWAN
+    default 64
+    help
+        Configure the ADR ACK limit. `ADRACKReq` bit (FOpts) is set to 1 once
```

Comment:
As mentioned above, I think this is too low level. This should reflect what the ADR ACK limit is